lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -105,11 +124,16 @@ class TagsBlockingFirstPaint extends Gatherer {
       return tags.reduce((prev, tag) => {
         const request = requests[tag.url];
         if (request && !request.isLinkPreload) {
+          const nonMatchingMediaChangeTimes = (tag.mediaChanges || [])
+            .filter(change => !change.matches)
+            .map(change => change.time);
+          const earliestNonBlockingTime = Math.min(...nonMatchingMediaChangeTimes);
+
           prev.push({
             tag,
             transferSize: request.transferSize || 0,
             startTime: request.startTime,
-            endTime: request.endTime,
+            endTime: Math.min(request.endTime, request.startTime + earliestNonBlockingTime / 1000),
```

Comment:
can you extract `request.startTime + earliestNonBlockingTime / 1000)` into a variable above?

Reply:
done